src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSpecV2.groovy

```diff
@@ -1008,6 +1018,42 @@ misconfigured"
         sectionsToVerifyPresence << [["meters"], ["meters", "groups"], ["meters", "groups", "rules"]]
     }
 
+    @Unroll
+    @Tidy
+    @Tags([VIRTUAL, LOW_PRIORITY])
+    def "Able to validate switch using #apiVersion API when GRPC is down"() {
+        given: "Random switch without LAG feature enabled"
+        def aSwitch = topology.getSwitches().find {
```

Comment:
aSwitch means `A switch` wich can break ISL, or it is just article?

Reply:
Just an article. I'm trying to avoid abbreviations (when remember about it).